### DISCUSSION TITLE
Range stuck at unknown

### DISCUSSION THREAD

**Discussion by Depechie** [pypolestar/polestar_api#126] [Q&A]
Any others have the same issue as I have? The integration seems to be working fine but the range value is reporting Unknown for a few days now...

![image](https://github.com/leeyuentuen/polestar_api/assets/351693/8b0ae93d-cadb-47b0-90a6-392ab48d079e)

**Comment by leeyuentuen** [comment]
which version are you running?

**Reply by Depechie** [reply]
1.6.3 ![image](https://github.com/leeyuentuen/polestar_api/assets/351693/f7313eee-c4bd-4c72-8253-7b9c4e86a10b)

**Comment by leeyuentuen** [comment]
i've just merge dev to master with version 1.6.10

there was before few dev changes
https://github.com/leeyuentuen/polestar_api/releases


could be solved in new version

**Reply by Depechie** [reply]
And miraculously today it shows the range again.
I guess API hiccup...
